dm/templates/iam_member/iam_member.py

```diff
@@ -13,28 +13,58 @@
 # limitations under the License.
 """ This template creates an IAM policy member. """
 
+from hashlib import sha1
+
 
 def generate_config(context):
     """ Entry point for the deployment resources. """
 
-    project_id = context.properties.get('projectId', context.env['project'])
+    folder_id = context.properties.get('folderId')
+    org_id = context.properties.get('organizationId')
+    project_id = context.properties.get('projectId')
+    if not folder_id and not org_id:
+        project_id = context.env['project']
```

Comment:
This means `context.properties.get('projectId')` is never used. Line 25-26 should be removed.
Line 24 should be:
`    project_id = context.properties.get('projectId', context.env['project'])`

Reply:
Fixed.